metricbeat/module/ceph/osd_tree/_meta/fields.yml

```diff
@@ -0,0 +1,57 @@
+- name: osd_tree
+  type: group
+  description: >
+    ceph osd tree info
+  fields:
+    - name: id
+      type: long
+      description: >
+        osd or bucket node id
+    - name: name
+      type: string
+      description: >
+        osd or bucket node name
+    - name: type
+      type: string
+      description: >
+        osd or bucket node type, illegal type include osd, host, root etc.
+    - name: type_id
+      type: long
+      description: >
+        osd or bucket node typeID
+    - name: children
+      type: string
+      description: >
+        bucket children list, seperate by comma.
+    - name: crush_weight
+      type: float
+      description: >
+        osd node crush weight
+    - name: depth
+      type: long
+      description: >
+        node depth
+    - name: exists
+      type: long
```

Comment:
this could be a boolean

Reply:
hi, I have update the fields.yml and the changelog.